source/manuals/programming-languages/editorconfig

```diff
@@ -1,15 +1,50 @@
-[*.java]
+# http://EditorConfig.org
+# https://gds-way.cloudapps.digital/manuals/programming-languages/editorconfig
+# Copy into project root and rename to .editorconfig
+
+# uncomment if top-most EditorConfig file
+# root = true
+
+[*]
 indent_style: space
-indent_size: 4
-end_of_line: lf
-charset: utf-8
+charset = utf-8
+end_of_line = lf
 insert_final_newline: true
+trim_trailing_whitespace = true
+
+[*.java]
+indent_size: 4
 continuation_indent_size: 8
+/* this wasn't explicitly set, so setting to false until told otherwise */
+trim_trailing_whitespace = false
+
+[*.{scss,erb,html,css}]
+indent_size = 2
 
 [*.{js,json}]
-charset = utf-8
-end_of_line = lf
-indent_style = space
 indent_size = 2
-insert_final_newline = true
-trim_trailing_whitespace = true
+
+[*.py]
+indent_size = 4
+max_line_length = 119
+
+[**.rb]
```

Comment:
Is there a reason this is the only file type where options are applied to files under directories? Should the others do this too? Should this do the same as the others?

Reply:
Good catch, updated to match the others